### DISCUSSION TITLE
If indexer order changes, rename episodes if possible

### DISCUSSION THREAD

**Discussion by mglaske** [pymedusa/Medusa#8699] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
Currently, for some shows, entries to tvdb are incorrect, and new episodes are added to current seasons, then later moved to new seasons.  This leaves orphaned/archived files with no description in the previous season, and sometimes starts downloading new copies for the new season.

**Describe the solution you'd like**
When an update happens, and the season/episode architecture pulled from tvdb is different than medusa, then compare show names (main keywords possibly) for episodes that have been removed or added, and rename the files appropriately.

**Describe alternatives you've considered**
Currently, people watching complain, and I go in and manually straighten things out.  It's quite difficult to get medusa to 'forget' the old episodes.

**Comment by OmgImAlexis** [comment]
Only issue with this is someone vandalising a series which would cause all your nicely named files to be rewritten.

**Comment by mglaske** [comment]
Yea, definitely!  I figured it would be an option you could turn on or off as needed..   Seems like it should be pretty safe if it’s based loosely upon the episode name..
